src/components/dialog/GitLogDialog.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <default-dialog dialog-key="GitLog" data-cy="git-log-dialog">
+    <template v-slot:title>
+      <q-icon color="primary" name="fa-brands fa-git-alt" />
+      {{ $t('page.modelizer.git.log.title') }}
+    </template>
+    <template v-slot:default>
+      <div ref="scrollTargetRef" style="max-height: 250px; overflow: auto;">
+        <q-infinite-scroll
+          @load="onLoad"
+          :offset="250"
+          :scroll-target="scrollTargetRef"
+        >
+          <q-list>
```

Comment:
indent issue (check :scroll-target above, they should be align)

Reply:
done